src/app/views/users/UsersList.jsx

```diff
@@ -0,0 +1,174 @@
+
+export const mapUserToState = (rootPath, user) => {
+    try {
+        const id = user.email;
+        const title = `${user.forename} ${user.lastname}`;
+        const details = [user.email];
+        const url = `${rootPath}/users/${id}`;
+        return { ...user, id, title, details, url };
+    } catch (error) {
+        const notification = {
+            type: "warning",
+            message: "Error mapping users to state",
+            isDismissable: true,
+            autoDismiss: 3000,
+        };
+        notifications.add(notification);
+        console.error("Error mapping users to state: ", error);
+        log.event("Error mapping users to state", log.error(error));
+        return false;
+    }
+};
+
+const propTypes = {
+    rootPath: PropTypes.string.isRequired,
+    dispatch: PropTypes.func.isRequired,
+    children: PropTypes.element,
+    params: PropTypes.shape({
+        userID: PropTypes.string,
+    }).isRequired,
+    loggedInUser: PropTypes.object,
+    users: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
+};
+
+export const UsersList = (props) => {
+    const [isFetchingUsers, setIsFetchingUsers] = useState(false);
+    const isAdmin = auth.isAdmin(props.loggedInUser);
+    const [filteredUsers, setFilteredUsers] = useState(null);
+
+    useEffect(() => {
+        if (!isAdmin && !props.params.userID) {
+            props.dispatch(replace(`${props.rootPath}/users/${props.loggedInUser.email}`));
+        }
+        getAllUsers(props.dispatch, props.rootPath, setIsFetchingUsers);
+    }, []);
+
+    const handleSearchInput = event => {
+        const searchTerm = event.target.value.toLowerCase();
+        const filteredUsers = props.users.filter(
+            user => user.title.toLowerCase().search(searchTerm) !== -1 || user.email.toLowerCase().search(searchTerm) !== -1
+        );
+        setFilteredUsers(filteredUsers);
+    };
+
+    return (
+        <div>
+            <div className="grid grid--justify-space-around">
+                <div className="grid__col-9">
+                    <BackButton fill="#000000" url={`${props.rootPath}/collections`} />
+                    <div className="grid grid--align-baseline">
+                        <div className="grid__col-1">
+                            <h1>Users</h1>
+                        </div>
+                        <div className="grid__col-1">
+                            <a href="#">Create new user</a>
+                        </div>
+                    </div>
+                    <div className="grid">
+                        <div className="grid__col-4">
+                            <Input id="search-content-types" placeholder="Search user by name or email" onChange={handleSearchInput} />
+                        </div>
+                    </div>
+                    <SimpleSelectableList rows={filteredUsers ? filteredUsers : props.users} showLoadingState={isFetchingUsers} />
+                </div>
+            </div>
+            {props.children}
```

Comment:
Do we need this, couldn't see it in the propTypes so not sure what this is doing - if not we do remove it then the outer div can go too I think.

Reply:
Missed this, was left from the UsersController should have been taken out.